src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -96,6 +96,7 @@
     private static final String XPATH_DESIGN_THEME = "/ObjectMetadata/OutputChannels/DIFTcom/DesignTheme";
     private static final String XPATH_DESIGN_LAYOUT = "/ObjectMetadata/OutputChannels/DIFTcom/DesignLayout";
     private static final String XPATH_PUSH_NOTIFICATION_COHORT = "/ObjectMetadata/OutputChannels/DIFTcom/pushNotification";
+    private static final String XPATH_PUSH_NOTIFICATION_TEXT = "/doc/lead/push-notification/ln";
```

Comment:
The path is wrong, it should be with: push-notification-text instead of push-notification.